Lib/collections/__init__.py

```diff
@@ -551,7 +550,8 @@ def most_common(self, n=None):
         # Emulate Bag.sortedByCount from Smalltalk
         if n is None:
             return sorted(self.items(), key=_itemgetter(1), reverse=True)
-        return _heapq.nlargest(n, self.items(), key=_itemgetter(1))
+        import heapq
+        return heapq.nlargest(n, self.items(), key=_itemgetter(1))
```

Comment:
I don't think this change is reasonable.  You're micro-optimizing the expense of writing normal maintainable code.